app/src/main/res/layout/fragment_transaction_details.xml

```diff
@@ -183,7 +183,7 @@
             android:layout_width="match_parent"
             android:layout_weight="1"
             android:layout_marginLeft="@dimen/default_margin"
-            android:layout_marginRight="20dp"
+            android:layout_marginRight="4dp"
```

Comment:
I don't know if this makes it more readable when you have to add different margins instead of explicitly define one value.

Reply:
Otherwise the confirm button doesn't have the right margin when the reject button is gone.

Reply:
This needed to be changed because the other button can also be gone :-(